x-pack/plugins/security_solution/public/common/components/exceptions/exceptionable_endpoint_fields.json

```diff
@@ -17,6 +17,13 @@
   "Target.process.pe.original_file_name",
   "Target.process.pe.product",
   "Target.process.pgid",
+  "Target.process.thread.Ext.start_address_details.allocation_type",
+  "Target.process.thread.Ext.start_address_bytes_disasm_hash",
+  "Target.process.thread.Ext.start_address_allocation_offset",
+  "Target.process.thread.Ext.start_address_details.allocation_size",
+  "Target.process.thread.Ext.start_address_details.region_size",
+  "Target.process.thread.Ext.start_address_details.region_protection",
+  "Target.process.thread.Ext.start_address_details.memory_pe.imphash",
```

Comment:
```suggestion
  "Target.process.thread.Ext.start_address_details.memory_pe.imphash",
  "Target.process.thread.Ext.start_address_bytes",
  "process.executable",
  "process.name",
  "Target.process.executable",
  "Target.process.name",
  "process.Ext.code_signature", /* can we allow nested only? */
```

Reply:
@gabriellandau can you clarify what do you mean with `/* can we allow nested only? */` ?

Reply:
code_signature should be a `nested` comparison so "trusted": true can be included, keeping users from accidentally trusting untrusted signatures:
```
                "code_signature": [
                    {
                        "exists": true,
                        "status": "trusted",
                        "subject_name": "Microsoft Windows",
                        "trusted": true
                    }
                ],
```

Untrusted:
```
            "code_signature": [
                {
                    "exists": true,
                    "status": "errorCode_80092003",
                    "subject_name": "LogMeIn, Inc.",
                    "trusted": false
                }
            ],
```